ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageVideoAdPlayer.swift

```diff
@@ -72,6 +73,10 @@ class NewTabPageVideoPlayer {
 
       addMediaObservers()
 
+      if isPlaylistActive() {
+        PlaylistCoordinator.shared.mediaPlayer?.pause()
```

Comment:
This will only pause playback on the old playlist UI, you'd need to use `PlaylistCoordinator.shared.playerModel` for new UI, however you're likely better just adding a new method to `PlaylistCoordinator` called `pauseAllPlayback` or something that does a `FeatureList.kNewPlaylistUI.enabled` check and calls pause on `playerModel` in the new UI and pause on `mediaPlayer` on the old UI

Reply:
Fixed